### ISSUE TITLE
bug: Deno panicked when compiling executable

### ISSUE THREAD

**Issue by TillaTheHun0** [denoland/deno#15705]
Tried to compile an executable and received a panic.

## Reproduce

> This repo can be checked out with Gitpod, if you'd like a completely sandboxed environment to reproduce this

- Clone https://github.com/hyper63/hyper
- `cd images/nano`
- Run `RUST_BACKTRACE=1 make compile-linux`

Expect: an executable is built and output to the `./build/hyper-x86_64-unknown-linux-gnu`
Actual: Deno panics and does not output the executable

Backtrace:
```
============================================================
Deno has panicked. This is a bug in Deno. Please report this
at https://github.com/denoland/deno/issues/new.
If you can reliably reproduce this panic, include the
reproduction steps and re-run with the RUST_BACKTRACE=1 env
var set and include the backtrace in your report.

Platform: linux x86_64
Version: 1.25.0
Args: ["deno", "compile", "--target=x86_64-unknown-linux-gnu", "--allow-env", "--allow-read", "--allow-write=__hyper__,/tmp/hyper", "--allow-net", "--unstable", "--no-check=remote", "--output", "./build/hyper-x86_64-unknown-linux-gnu", "mod.js"]

thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/eszip-0.25.0/src/v2.rs:512:60
stack backtrace:
   0: rust_begin_unwind
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/panicking.rs:142:14
   2: core::panicking::panic
             at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/panicking.rs:48:5
   3: eszip::v2::EszipV2::from_graph::visit_module
   4: eszip::v2::EszipV2::from_graph::visit_module
   5: eszip::v2::EszipV2::from_graph::visit_module
   6: eszip::v2::EszipV2::from_graph
   7: deno::compile_command::{{closure}}
   8: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
   9: deno::main::{{closure}}
  10: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  11: deno_runtime::tokio_util::run_local
  12: deno::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
make: *** [Makefile:14: compile-linux] Error 1
```

**Comment by bartlomieju** [comment]
This has been fixed in https://github.com/denoland/deno/commit/c3e48cba184f2f8aaf3d30196b674c8a7dd8449b and will be released in v1.25.1

**Comment by TillaTheHun0** [comment]
That was fast, thanks for the update and swift solution! 🦕
